0.Preprocess_Data/scripts/3.data_aggregation.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import numpy as np
+import pandas as pd
+
+# In[2]:
+
+
+# Parameters
+cell_type = "SHSY5Y"
+aggregation = True
+nomic = True
+
+
+# In[ ]:
+
+
+if aggregation == True & nomic == True:
+    aggregated_data_path = pathlib.Path(
+        f"../data/{cell_type}_preprocessed_sc_norm_aggregated_nomic.parquet"
+    )
+elif aggregation == False & nomic == True:
+    aggregated_data_path = pathlib.Path(
+        f"../data/{cell_type}_preprocessed_sc_norm_nomic.parquet"
+    )
+elif aggregation == True & nomic == False:
```

Comment:
Similarly to the above:
```suggestion
elif aggregation and not nomic:
```